UPGRADE.md

```diff
@@ -2666,6 +2666,28 @@ const ReviewEditToolbar = (props: ToolbarProps<Review>) => {
 };
 ```
 
+Since the `<Toolbar>` component is not cloned anymore by react-admin layouts, `submitOnEnter` property is not passed down to the `SaveButton` from parent components. So, if you rely on this behaviour, you should pass this prop yourself.
```

Comment:
I think you should be more explicit and add a section titled 

> `<SimpleForm>` and `<TabbedForm>` don't support `submitOnEnter` anymore

and mention tat it also concerns Tabbedform